tools/wt_verify/wt_verify.py

```diff
@@ -172,6 +237,129 @@ def parse_output():
     return output
 
 
+def show_page_distribution(filename, data, bar_width=1):
```

Comment:
For this one, did you figure out what the `cell_type` was that kept showing up on the chart?

Reply:
Yes, after checking the output of `dump_blocks,` it corresponds to `value_ovfl`. I have updated the `cell_type` tag to the actual cell type in c1c605526c72473703afe7003b3e39c9e48244ec.